README.md

```diff
@@ -0,0 +1,14 @@
+# Spam Watch API
+
+API server for Spam Watch Bot
```

Comment:
SpamWatch is written together without a space
The API is also for everyone, not just the bot so you can remove the bot part